spec/rubocop/formatter/offense_count_formatter_spec.rb

```diff
@@ -20,14 +20,14 @@ module Formatter
         context 'when no offenses are detected' do
           let(:offenses) { [] }
           it 'does not add to offense_counts' do
-            expect { finish }.not_to change { formatter.offense_counts }
+            expect { finish }.not_to(change { formatter.offense_counts })
```

Comment:
Maybe we should add some exceptions (or simply disable this cop here) as I prefer the original version of the code. DSL make heavy use of this, aiming to produce a more natural code.